XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,41 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
+
+		//Navigate to a URL and perform a long press on the non-highlighted address bar
+		navigator.goto(URLBarOpen)
+		app.textFields["address"].typeText("www.google.com\n")
+		app.textFields["url"].tap()
+		app.textFields["address"].press(forDuration: 2)
+
+		//Ensure that long press on address bar brings up a menu with Select All, Select, Paste, and Paste & Go
+		XCTAssertTrue(app.menuItems["Select All"].exists)
+        XCTAssertTrue(app.menuItems["Select"].exists)
+        XCTAssertTrue(app.menuItems["Paste"].exists)
+        XCTAssertTrue(app.menuItems["Paste & Go"].exists)
+
+        //Tap on Select option and make sure Copy, Cut, Paste, and Look Up are shown
+        app.menuItems["Select"].tap()
```

Comment:
When tapping on 'Select' those options are shown and also an arrow > with more options, that should be checked too, is there where the Paste&Go option is in this case and when Selecting All.

Also, take into account that the options are different for iPad in each case. I would confirm if that's the case and if not open a bug but anyway the test needs to be modified or it will fail when running on iPad device